test/util/test_runners/integrations_tests_runner.py

```diff
@@ -50,42 +51,41 @@ def run_tests(self, test_modules: list):
     def start_env(self):
         """
             set up the test environment
```

Comment:
Test environment setup will:
1. ...
2. ...

Reply:
on it